### PR TITLE
Run a workspace!

### PR DESCRIPTION
**Pull request by acron0**
Now downloads data from S3, unzips if applicable, and drops the result on the ground.

### CODE REVIEW

File: src/witan/workspace/data.clj [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  [schema data]
+  (let [coerce-data-fn
+        (coerce/coercer schema
+                        coerce/string-coercion-matcher)]
+    (coerce-data-fn data)))
+
+(defn apply-row-schema
+  [col-schema data]
+  (let [row-schema (make-row-schema col-schema)]
+    (map #((partial (fn [s r] (record-coercion s r)) row-schema) %) data)))
```
Comment (otfrom) [inline src/witan/workspace/data.clj]: Won't `(map (partial record-coercion row-schema) data)` work?

or even get rid of `record-coercion`

`(map (coerce/coercer row-schema coerce/string-coercion-matcher) data)` but possibly not as `record-coercion` is used in other places.

Comment (acron0) [inline src/witan/workspace/data.clj]: This is actually lifted from witan.models.demography, [here](https://github.com/MastodonC/witan.models.demography/blob/master/src-cli/witan/models/load_data.clj#L41), so I'm not hugely confident in adjusting it...

**Comment by otfrom** [comment]
other than apply-row-schema :shipit:

File: src/witan/workspace/data.clj (hunk 1 above)
Comment (otfrom) [inline src/witan/workspace/data.clj]: create a test for apply-row-schema then change it. to `(map (partial record-coercion row-schema) data)`

**Comment by otfrom** [comment]
:shipit:

**Pull request merged**